assets/scripts/components/algolia.js

```diff
@@ -155,16 +165,28 @@ function loadInstantSearch(asyncLoad) {
 
             const handleSearchbarSubmitClick = () => {
                 if (aisSearchBoxInput.value) {
+                    sendSearchRumAction(search.helper.state.query);
                     window.location.pathname = searchPathname;
```

Comment:
does sending the query refresh the page? is that why we need to update the window pathname?